src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentService.kt

```diff
@@ -77,11 +77,15 @@ class CodyAgentService(project: Project) : Disposable {
       }
 
       agent.client.onOpenUntitledDocument = Function { params ->
-        val uri = URI.create(params.uri).withScheme("file")
-        if (CodyEditorUtil.createFileIfNeeded(project, uri, params.content) == null)
+        val uri = URI.create(params.uri)
+        if (!uri.path.contains("/")) {
+          return@Function false
+        }
+        val fileUri = uri.withScheme("file")
```

Comment:
I know this is not new, but this seems really dodgy to me. When do we get URIs that aren't `file:`? When would it be OK to take on of those and say randomly, boom, now you're a file path!

Reply:
In fact we always get a files which does not start with `file://` there.
They start with `untitled://` which is VSCode-specific way to say that files does not exist on the disk yet.
I should have added a comment. @mkondratek can you fix my omission and add it now?